packages/ui/react-ui-editor/src/components/Markdown/markdownTheme.ts

```diff
@@ -174,6 +179,7 @@ export const markdownTheme = {
   }, {}),
 };
 
+// TODO(burdon): Is this dark mode only?
```

Comment:
No, there is a comment where this is used that remarks on same. I think ‘dark’ can probably be removed.